### PR TITLE
add a note that Rancher does not clean up the builtin PSPs

### PR DESCRIPTION
**Pull request by jiaqiluo**
Issue: https://github.com/rancher/rancher/issues/40032

add a note that Rancher does not clean up the builtin PSPs

### CODE REVIEW

File: docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/create-pod-security-policies.md [hunk 1]
```diff
@@ -40,6 +40,12 @@ This policy is a relaxed version of the `restricted-noroot` policy, with almost
 
 This policy is equivalent to running Kubernetes with the PSP controller disabled. It has no restrictions on what pods can be deployed into a cluster or project.
 
+:::note important
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/create-pod-security-policies.md:43]:
```suggestion
:::important
```

File: docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/create-pod-security-policies.md [hunk 2]
```diff
@@ -40,6 +40,12 @@ This policy is a relaxed version of the `restricted-noroot` policy, with almost
...
+
+Rancher does not delete those default PSPs from your cluster automatically. You can delete them if they are no longer needed.
```
Comment (martyav) [inline docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/create-pod-security-policies.md:45]:
```suggestion
Rancher doesn't automatically delete default PSPs from your cluster. You must manually delete them if they're no longer needed. 
```

**Review by martyav — CHANGES_REQUESTED** [review]

File: docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/create-pod-security-policies.md (hunk 1 above)
Comment (btat) [reply docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/create-pod-security-policies.md]: While `:::important` works, it's an undocumented/legacy admonition and may break in the future. The original syntax here is correct. 

^@jiaqiluo

Comment (jiaqiluo) [reply docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/create-pod-security-policies.md:43]: Good to know. 
I reverted it to the original.

File: docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/create-pod-security-policies.md [hunk 3]
```diff
@@ -40,6 +40,12 @@ This policy is a relaxed version of the `restricted-noroot` policy, with almost
 
 This policy is equivalent to running Kubernetes with the PSP controller disabled. It has no restrictions on what pods can be deployed into a cluster or project.
 
+:::note important
+
+Rancher doesn't automatically delete default PSPs from your cluster. You must manually delete them if they're no longer needed.
```
[the remark this answers is not in the document]
Comment (jiaqiluo) [reply docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/create-pod-security-policies.md:45]: Rancher creates those 3 PSPs when the PSP feature is enabled on the cluster but does not remove them if the PSP feature is disabled later.   If the PSP feature is never enabled, those 3 PSPs will not be created. 

Can you help rephrase the sentence?

[the remark this answers is not in the document]
Comment (btat) [reply docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/create-pod-security-policies.md:45]:
```suggestion
When disabling PSPs, default PSPs are **not** automatically deleted from your cluster. You must manually delete them if they're no longer needed.
```

Maybe something like this? In a follow-up PR I could convert `disabling PSPs` and `manually delete` to link out to new sections with steps to do those.

[the remark this answers is not in the document]
Comment (jiaqiluo) [reply docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/create-pod-security-policies.md]: Cool! The PR is updated.

**Review by btat — APPROVED** [review]

**Review by martyav — APPROVED** [review]

**Pull request merged**